src/pytorch_lightning/demos/boring_classes.py

```diff
@@ -107,43 +108,43 @@ def training_step(self, batch, batch_idx):
         loss = self.loss(batch, output)
         return {"loss": loss}
 
-    def training_step_end(self, training_step_outputs):
+    def training_step_end(self, training_step_outputs: Any) -> Any:
         return training_step_outputs
 
-    def training_epoch_end(self, outputs) -> None:
+    def training_epoch_end(self, outputs: Any) -> None:
```

Comment:
Can you please check the type here as well? Should be the same as in the LightningModule class